doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst

```diff
@@ -9,6 +9,21 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+
+Controller v3393
+****************
+
+The following changes have been introduced in the v3393 of the controller:
+
+Changes
+=======
+
+* All necessary EBQ tests for claiming the QDID now pass.
+* The controller code in general has been significantly updated, which improves the performance on many levels.
+* Fixed an issue where the timing of the TXEN FEM pin was wrong, causing unwanted radio noise.
+* Correct timestamps are are now always returned when using the LE_READ_ISO_TX_SYNC HCI command.
```

Comment:
```suggestion
* Updated the controller code with major changes that improve performance on many levels.
* Updated all necessary EBQ tests for claiming the QDID.
* Fixed an issue where the timing of the TXEN FEM pin was wrong and would cause unwanted radio noise.
* Fixed timestamps that are returned when using the LE_READ_ISO_TX_SYNC HCI command.
```

Reply:
Changed to what you proposed, with one adjustment